freqtrade/rpc/rpc.py

```diff
@@ -460,3 +460,18 @@ def _rpc_whitelist(self) -> Dict:
                'whitelist': self._freqtrade.active_pair_whitelist
                }
         return res
+
+    def _rpc_edge(self) -> Dict:
+        """ Returns information related to Edge """
+        if not self._freqtrade.edge:
+            raise RPCException(f'Edge is not enabled.')
+
+        return [
+            {
+                'Pair': pair,
+                'Winrate': self._freqtrade.edge._cached_pairs[pair].winrate,
+                'Expectancy': self._freqtrade.edge._cached_pairs[pair].expectancy,
+                'Stoploss': self._freqtrade.edge._cached_pairs[pair].stoploss,
+            }
+            for pair in self._freqtrade.edge._cached_pairs
```

Comment:
@xmatthias I am sure you have a better idea for the above iteration :)

Reply:
it's not an iteration but a list comprehension - which is quite performant - better than an iteration..

however, using `for key, value in d.items():` might be better - so you can use `value.winrate` instead of querying the dict ...